dataland-frontend/tests/e2e/fixtures/lksg/LksgPreparedFixtures.ts

```diff
@@ -42,6 +44,7 @@ function manipulateFixtureForOneLksgDataSetWithProductionSites(input: FixtureDat
   const twoProductionSites = [generateProductionSite(), generateProductionSite()];
   input.companyInformation.companyName = "one-lksg-data-set-with-two-production-sites";
   input.t.general.productionSpecific!.listOfProductionSites = twoProductionSites;
+  input.t.governance!.certificationsPoliciesAndResponsibilities!.sa8000Certification!.value = YesNo.Yes;
```

Comment:
Why is this suddenly needed in this story?

Reply:
There is a test that relies on checking a link and it fails if the value is not set to yes since then the link does not appear. I introduced this line to prevent flakiness with regard to the test data set. I think in the past it just worked by chance.